core/platforms/win32/winbareos-native.nsi

```diff
@@ -1770,7 +1739,6 @@ Function getDatabaseParameters
   strcmp $Upgrading "yes" skip
   strcmp $InstallDirector "no" skip
 
-${If} ${SectionIsSelected} ${SEC_DIR_POSTGRES}
   # prefill the dialog fields
   WriteINIStr "$PLUGINSDIR\databasedialog.ini" "Field 3" "state" $DbAdminUser
   WriteINIStr "$PLUGINSDIR\databasedialog.ini" "Field 4" "state" $DbAdminPassword
```

Comment:
While at it, we may want to move to something less sensitive than just str field so the password of postgres superuser is not shown as clear text in the dialog box?